src/components/stage-comments/modules/popup-com.js

```diff
@@ -0,0 +1,18 @@
+// function comment (item) {
+//     document.getElementById(item).addEventListener('click', () => {
+//         const data = {
+//             "item_id": item,
+//             "userName": item,
+//             "comment": item,
+//         }
+
+//         fetch('https://www.breakingbadapi.com/api/characters?limit=12&offset=0', {
+//             method: 'POST',
+//             body: JSON.stringify(data),
+//             headers: {'content-type': 'application/json; charset-UTF-8'},
+//         })
+//         .then((response) => response.json())
+//         .then((json) => console.log(json))
+//         .catch((err) => console.log(err));
+//     })
+// }
```

Comment:
- [ ] to make your code looks clean kindly remove the commented code from all of your pages including [addCharApi.js](https://github.com/erickhansx/capstone-javascript/pull/32/files#diff-9fedd86f195b22921fbd503c63c3b7e6b08d794147cbf61be767603b46f08ff0) , [index.js](https://github.com/erickhansx/capstone-javascript/pull/32/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556), [displayCharacters.js](https://github.com/erickhansx/capstone-javascript/pull/32/files#diff-183ef578a1989a2f9d302a565af109a0145091a5dcbf39dc1e7cdb525b1ddbbd) , popup-com.js etc ... 🙏